pkg/tests/integration/test_salt_user.py

```diff
@@ -170,3 +171,195 @@ def test_pkg_paths(
+                                f"user: {test_account.username}",
+                                "flags=['IGNORECASE']",
+                                "append_if_not_found=True",
+                            )
+                            assert ret.returncode == 0
+
+                            # change ownership of appropriate paths to user
+                            for _path in log_pkg_paths:
+                                chg_ownership_cmd = (
+                                    f"chown -R {test_account.username} {_path}"
+                                )
+                                ret = salt_call_cli.run(
+                                    "--local", "cmd.run", chg_ownership_cmd
+                                )
+                                assert ret.returncode == 0
+
+                            # restart the salt_master
+                            with salt_master.started():
+                                assert salt_master.is_running() is True
+
+                                # ensure some data in files
+                                log_files_list = [
+                                    "/var/log/salt/api",
+                                    "/var/log/salt/key",
+                                    "/var/log/salt/master",
+                                ]
+                                for _path in log_files_list:
+                                    log_path = pathlib.Path(_path)
+                                    assert log_path.exists()
+                                    with log_path.open("a") as f:
+                                        f.write("This is a log rotation test\n")
+
+                                # force log rotation
+                                logr_conf_file = "/etc/logrotate.d/salt"
+                                logr_conf_path = pathlib.Path(logr_conf_file)
+                                # assert logr_conf_path.exists()
+                                if not logr_conf_path.exists():
+                                    logr_conf_file = "/etc/logrotate.conf"
+                                    logr_conf_path = pathlib.Path(logr_conf_file)
+                                    assert logr_conf_path.exists()
+
+                                # force log rotation
+                                log_rotate_cmd = f"logrotate -f  {str(logr_conf_file)}"
+                                ret = salt_call_cli.run(
+                                    "--local", "cmd.run", log_rotate_cmd
+                                )
+                                assert ret.returncode == 0
+
+                                for _path in log_files_list:
+                                    log_path = pathlib.Path(_path)
+                                    str_log_path = str(log_path)
+                                    ret = salt_call_cli.run(
+                                        "--local", "cmd.run", f"ls -alh {str_log_path}"
+                                    )
```

Comment:
```suggestion
                                    ret = salt_call_cli.run(
                                        "--local", "cmd.run", f"ls -alh {str_log_path}"
                                    )
```
This is likely not needed right?
Since nothing is checked againt `ret`

Reply:
removed it and str_log_path assignment